### PR TITLE
rootfs-configs.yaml: Add bullseye-cros-ec config

### PR DESCRIPTION
**Pull request by mgalka**
Add bullseye based image configuration for cros-ec tests.

### CODE REVIEW

**Comment by gctucker** [comment]
Has it been built on staging?  I think that would be the next step.

**Comment by mgalka** [comment]
> Has it been built on staging? I think that would be the next step.

Yes, it was on Friday and I also ran a test job with the new rootfs for x86_64 on a device. The `unittest` package seems still to be uninstalled. I investigated it and it seems that `strip.sh` script ran by `kci_rootfs` removes the package `libpython3.9-stdlib` before it gets marked as "manual". I've just pushed a workaround for it. I'll give it a testing round.

**Comment by gctucker** [comment]
Have you tested your workaround on staging or with a LAVA manual resubmit?

File: config/rootfs/debos/scripts/strip.sh [hunk 1]
```diff
@@ -10,8 +10,7 @@ rm -rf /etc/localtime
 cp /usr/share/zoneinfo/Etc/UTC /etc/localtime
 
 
-UNNEEDED_PACKAGES=" libfdisk1"\
-" tzdata"\
+UNNEEDED_PACKAGES=" libfdisk1"
```
Comment (gctucker) [inline config/rootfs/debos/scripts/strip.sh:13]: What is the difference in ramdisk size?  Ideally this should be a parameter in rootfs-configs.yaml to remove it when it's not neeed.  Maybe we could run cros-ec tests with NFS instead in this case, as it has the full rootfs image before being stripped.

**Comment by mgalka** [comment]
Closing this PR as the fix has been split into:
https://github.com/kernelci/kernelci-core/pull/995
https://github.com/kernelci/kernelci-core/pull/996

**Pull request closed without merging**